django/db/migrations/operations/models.py

```diff
@@ -17,7 +17,7 @@ def _check_for_duplicates(arg_name, objs):
     for val in objs:
         if val in used_vals:
             raise ValueError(
-                'Found duplicate %s in CreateModel operation.' % arg_name
+                'Found duplicate %s in CreateModel operation: %s' % (arg_name, val)
```

Comment:
Suggestion: "Found duplicate value %s in CreateModel managers/fields/bases argument."